catalogbuilder/intakebuilder/builderconfig.py

```diff
@@ -15,7 +15,7 @@
 headerlist = ["activity_id", "institution_id", "source_id", "experiment_id",
                   "frequency", "realm", "table_id",
                   "member_id", "grid_label", "variable_id",
-                  "time_range", "chunk_freq","grid_label","platform","dimensions","cell_methods","standard_name","path"]
+                  "time_range", "chunk_freq","platform","dimensions","cell_methods","standard_name","path"]
```

Comment:
Is this file an example template for users to use?

e.g. with the `fre catalog build --config` option?

What is this file here doing?

Reply:
It's there because we don't a unified code base to parse the headers for no-config and config option. This will be a major change to the repo and I have documented the issue  and we can prioritize this for the next month. 

https://github.com/NOAA-GFDL/CatalogBuilder/issues/81

Reply:
Got it. Thanks, totally agree.